src/index.coffee

```diff
@@ -119,8 +131,8 @@ module.exports = class GeordiClient
 
   _updateEventDataFromParameterObject: (parameterObject, eventData = {}) ->
     # copy all string & numeric data across
-    for field in ["userID","subjectID","relatedID","errorCode","errorDescription","projectToken","serverURL","experiment","cohort","type","userSeq","sessionNumber","eventNumber","userAgent","clientIP"]
```

Comment:
OK i see where the list above came from now. Ignore my question about time fields above.